posthog/api/organization_member.py

```diff
@@ -39,24 +39,21 @@ def has_object_permission(self, request: Request, view, membership: Organization
 class OrganizationMemberSerializer(serializers.ModelSerializer):
     user = UserBasicSerializer(read_only=True)
     is_2fa_enabled = serializers.SerializerMethodField()
+    has_social_auth = serializers.SerializerMethodField()
 
     class Meta:
         model = OrganizationMembership
-        fields = [
-            "id",
-            "user",
-            "level",
-            "joined_at",
-            "updated_at",
-            "is_2fa_enabled",
-        ]
+        fields = ["id", "user", "level", "joined_at", "updated_at", "is_2fa_enabled", "has_social_auth"]
         read_only_fields = ["id", "joined_at", "updated_at"]
 
     def get_is_2fa_enabled(self, instance: OrganizationMembership) -> bool:
         # If we add other forms of 2FA we need to use default_device here instead
         # But not using that here as it increased the number of queries we did by a lot
         return len(instance.user.totpdevice_set.all()) > 0
 
+    def get_has_social_auth(self, instance: OrganizationMembership) -> bool:
+        return instance.user.social_auth.exists()
```

Comment:
It seems that this blows up the number of queries we are making. How else can I include this info on the members api? We need it because:

- Existing users all have `is_email_verified == Null`
- If they log in with SSO, they DON'T need to verify their email (SSO is verification in and of itself)
- We don't want to ask them to verify if they use SSO. 
- We don't want to show, on the org members page, that they haven't verified if they use SSO. 
- So we need to know if they have SSO for their account.

Reply:
Seems like one extra query from the tests? Is that the one you're worried about?

If yes, do not worry, these are some of the least perf-based endpoints, and these relational postgres queries on models are very cheap.

Reply:
It's one extra query per member. So if there are 10 members it's 10 extra queries. At least that's what it seemed to be doing..

Reply:
Ah I see, yeah that's annoying, you can use `prefetch_related` to convert that to a single query instead.

Here's an example: https://github.com/PostHog/posthog/blob/master/posthog/api/organization_member.py#L91